rollup.config.js

```diff
@@ -62,7 +62,16 @@ export default {
 
 	<link rel='icon' type='image/png' href='favicon.png'>
 	${css}
-	<link rel='manifest' href='manifest.json'>
+  <link rel='manifest' href='manifest.json'>
+
+
+  <script>
+  window.addEventListener('load', () => {
+    if ('serviceWorker' in navigator) {
```

Comment:
Service-woker - better to connect in html file.
In svelte it can be put into a component. 
But none of the existing ones are suitable.
The problem is that it has to be plugged into the root.
Or, the service-woker will not connect.

Reply:
Connecting to index.html is the easiest and clearest way.

Reply:
By putting it in `onMount()` it seems to work fine? It's a cleaner way to put it, I prefer to keep the index code in the rollup config as minimal as possible.